tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -644,6 +645,7 @@ def fit(self,
         ag_args_fit = kwargs['ag_args_fit']
         ag_args_ensemble = kwargs['ag_args_ensemble']
         excluded_model_types = kwargs['excluded_model_types']
+        fold_fitting_strategy = kwargs['fold_fitting_strategy']
```

Comment:
should not be a parameter, instead it should be a value option in `ag_args_ensemble`.